code/game/objects/items/melee/baton.dm

```diff
@@ -437,7 +437,7 @@
 	var/throw_stun_chance = 35
 	var/obj/item/stock_parts/cell/cell
 	var/preload_cell_type //if not empty the baton starts with this type of cell
-	var/cell_hit_cost = 1000
+	var/cell_hit_cost = STANDARD_CELL_CHARGE
```

Comment:
```suggestion
	var/cell_hit_cost = STANDARD_CELL_CHARGE * 0.1
```

Here & all other cases. Previously the maximum capacity of a cell was 10000(a.k.a previous value of  STANDARD_CELL_CHARGE) so 1000 is 0.1 of it